general/g.gisenv/g.gisenv.html

```diff
@@ -145,7 +145,7 @@ <h2>NOTES</h2>
 By default the GRASS variables are stored in <em>gisrc</em> file
 (defined by environmental
 variable <em>GISRC</em>). If <b>store=mapset</b> is given than the
-variables are stored in <tt>$GISDBASE/$LOCATION_NAME/$MAPSET/VAR</tt>.
+variables are stored in <tt>$GISDBASE/$LOCATION_NAME/$MAPSET/VAR</tt> after the current GRASS session is closed.
```

Comment:
```suggestion
variables are stored in <tt>$GISDBASE/$LOCATION_NAME/$MAPSET/VAR</tt>
after the current GRASS session is closed.
```
(breaking long lines as per our HTML style guide)

Reply:
Maybe also mention "and stored in a temporary file during the current session."